adobe/spectrum-web-components
# fix: handle tap highlight color for android

## Description
Includes `-webkit-tap-highlight-color` in core CSS for both Classic and Express variants

## Related issue(s)
- fixes #2697

## Motivation and context
Prevent visible tap highlight in Android

## How has this been tested?

    1. In Android Chrome
    2. Go [here](https://tap-highlight--spectrum-web-components.netlify.app/)
    3. Tap things like Side Nav Items and Tab elements
    4. See that the blue background highlight does not display

## Types of changes

## Checklist
